docs/guides/administrator/gcp-setup.md

```diff
@@ -0,0 +1,162 @@
+
+# How to set up a GCP project for Opstrace
+
+This is a detailed walk-through showing how to set up a fresh GCP project for Opstrace.
+
+## Step 1: create a project
+
+Start by creating a fresh project with the [GCP UI](https://console.cloud.google.com/projectcreate).
+In addition to using a good project name (which you can change later), be sure to choose an expressive **project ID** on that screen (as the UI says: **it cannot be changed later**).
```

Comment:
Note to self: we recommend to pick the ID first, and then set the name to the ID to prevent confusion?

Reply:
done